.test-infra/jenkins/job_PostCommit_Java_Nexmark_Flink.groovy

```diff
@@ -43,6 +43,7 @@ NoPhraseTriggeringPostCommitBuilder.postCommitJob('beam_PostCommit_Java_Nexmark_
       switches('-Pnexmark.runner=":runners:flink:1.9"' +
               ' -Pnexmark.args="' +
               [NexmarkBigqueryProperties.nexmarkBigQueryArgs,
+               '--runner=FlinkRunner',
```

Comment:
Does this affect the runner used or just the table name? I'm asking because I believe the runner used is TestFlinkRunner (while the existing table is uses the former runner name).
@mxm

Reply:
It affects both actually and no, TestFlinkRunner is not used for Nexmark since this is not a 'Test' but a real 'execution'

Reply:
`TestFlinkRunner` is used to set the FlinkRunner's `shutdownSourcesOnFinalWatermark` pipeline option. In the regular `FlinkRunner`, streaming pipelines never shuts down which causes the tests to block here. When sources shut down checkpointing stops working in Flink, that's why we never shut down in production systems. See also https://jira.apache.org/jira/browse/FLINK-2491

Reply:
You can use `--shutdownSourcesOnFinalWatermark` here.

Reply:
Adding the above pipeline option will unblock streaming.